internal/signer/linux/pkcs11/pkcs11_test.go

```diff
@@ -34,3 +40,79 @@ func TestParseHexStringFailure(t *testing.T) {
 		t.Error("Expected error but got nil")
 	}
 }
+
+func TestEncryptRSA(t *testing.T) {
+	key, _ := Cred("/usr/local/lib/softhsm/libsofthsm2.so","0x268c8a20","Demo Object","0000")
+	msg := "Plain text to encrypt"
+	bMsg := []byte(msg)
+	ciphertext, err := key.EncryptRSA(sha256.New(), bMsg)
+	if err != nil {
+		t.Fatalf("EncryptRSA error: %q", err)
+	}
+	if ciphertext == nil {
+		t.Errorf("EncryptRSA error: empty ciphertext")
+	}
+}
+
+func TestCredLinux(t *testing.T) {
+	// parameters from https://paste.googleplex.com/5330692178182144 (slot from line 46)
```

Comment:
Remove this comment.

Reply:
Instead we will want to integrate that script into the repo, so it can be used by the CI.

Reply:
I removed the comment. Should I make a separate file for the script inside the repo?

Reply:
Let's introduce the script in a separate PR. Make a new folder called `scripts` and add it as `softhsm_setup.sh`

Reply:
Okay